frontend/src/locales/scripts/en.json5

```diff
@@ -5,7 +5,7 @@
   },
   hero: {
     subtitle: "Explore more than 700 million creative works",
-    description: "An extensive library of free stock photos, images, and audio, available for free use.",
+    description: "An extensive library of free stock photos, images, and audio track, available for free use.",
```

Comment:
The word "track" was not needed here as we are referring to the entire body of audio works. Please revert.